hack/jenkins/test-image/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM golang:1.5.3
+FROM golang:1.6
```

Comment:
nit: there is difference between `golang:1.6` and `golang:1.6.0` which I suppose we will target.
Although I think you know, `golang:1.6` will point to `go1.6.1` when released and so on. 
Not sure if important, but `golang:1.4` caused issues earlier